functions

```diff
@@ -455,6 +455,18 @@ function workaround_start_sshd {
     fi
 }
 
+function workaround_git_config {
+    # Workaround: fatal: unable to auto-detect email address (got 'stack@runner-bkqvjp4m-project-21118777-concurrent-0.(none)')
+    # Might not be the best place to put this, but it is causing control_host_bootstrap to fail.
+    git config --global user.name "Kayobe Automation"
+    git config --global user.email "kayobe-automation@example.org"
```

Comment:
Does this require --global?

Reply:
We could possibly get away with setting this for every git repository we clone, but I thought it represented a sensible default. I didn't actually expect a pip install to need this to be set and I didn't want to expose this complication to the user. We do override this to a user controlled value for any commits we actually make:

https://github.com/stackhpc/kayobe-automation/blob/main/functions#L269

Reply:
I think I'd be a bit surprised if I ran this locally then found I was suddenly making commits as someone called "Kayobe Automation" :)

Given that it's all scripted, if it's possible to set the config on a per-repo basis, I'd say that would be less invasive.